spec/features/cancelling_a_whole_plate_spec.rb

```diff
@@ -26,116 +26,113 @@
     json :plate, barcode_number: example_plate.labware_barcode.number, uuid: plate_uuid, state: 'passed'
   end
 
-  def expect_state_change_create
-    expect_api_v2_posts(
-      'StateChange',
-      [
-        {
-          contents: %w[
-            A1
-            B1
-            C1
-            D1
-            E1
-            F1
-            G1
-            H1
-            A2
-            C2
-            D2
-            E2
-            F2
-            G2
-            H2
-            A3
-            B3
-            C3
-            D3
-            E3
-            F3
-            G3
-            H3
-            A4
-            B4
-            C4
-            D4
-            E4
-            F4
-            G4
-            H4
-            A5
-            B5
-            C5
-            D5
-            E5
-            F5
-            G5
-            H5
-            A6
-            B6
-            C6
-            D6
-            E6
-            F6
-            G6
-            H6
-            A7
-            B7
-            C7
-            D7
-            E7
-            F7
-            G7
-            H7
-            A8
-            B8
-            C8
-            D8
-            E8
-            F8
-            G8
-            H8
-            A9
-            B9
-            C9
-            D9
-            E9
-            F9
-            G9
-            H9
-            A10
-            B10
-            C10
-            D10
-            E10
-            F10
-            G10
-            H10
-            A11
-            B11
-            C11
-            D11
-            E11
-            F11
-            G11
-            H11
-            A12
-            B12
-            C12
-            D12
-            E12
-            F12
-            G12
-            H12
-          ],
-          customer_accepts_responsibility: nil,
-          reason: 'Not required',
-          target_state: 'cancelled',
-          target_uuid: plate_uuid,
-          user_uuid: user_uuid
-        }
-      ]
-    )
+  let(:state_changes_attributes) do
+    [
+      {
+        contents: %w[
```

Comment:
`contents: WellHelpers.column_order`

Reply:
Turns out "B2" is missing from the set so I removed it from the response from WellHelpers.  Makes sense since it's marked as "failed" in the setup above.